benefits/in_person/views.py

```diff
@@ -91,20 +97,31 @@ def enrollment(request):
                     expiration_datetime=expiry,
                 )
                 event.save()
+                enrollment_analytics.returned_success(
+                    request, flow.group_id, enrollment_method=models.EnrollmentMethods.IN_PERSON
+                )
                 return redirect(routes.IN_PERSON_ENROLLMENT_SUCCESS)
 
             case Status.SYSTEM_ERROR:
+                enrollment_analytics.returned_error(
+                    request, str(exception), enrollment_method=models.EnrollmentMethods.IN_PERSON
+                )
                 sentry_sdk.capture_exception(exception)
                 return redirect(routes.IN_PERSON_ENROLLMENT_SYSTEM_ERROR)
 
             case Status.EXCEPTION:
+                enrollment_analytics.returned_error(
+                    request, str(exception), enrollment_method=models.EnrollmentMethods.IN_PERSON
+                )
                 sentry_sdk.capture_exception(exception)
                 return redirect(routes.IN_PERSON_SERVER_ERROR)
 
             case Status.REENROLLMENT_ERROR:
                 return redirect(routes.IN_PERSON_ENROLLMENT_REENROLLMENT_ERROR)
     # GET enrollment index
     else:
+        flow = session.flow(request)
+        eligibility_analytics.returned_success(request, flow, enrollment_method=models.EnrollmentMethods.IN_PERSON)
```

Comment:
I think in general, we should move away from sending events in response to `GET` requests where possible.

It becomes easier to spam the events if all you need to do is visit the URL. Especially for these types of events where we do already have a `POST`-back, I think ideally the event should be sent there.

Reply:
Ah that makes sense! Moved it to the `POST` section in e9f5ead8ea7e813c6f360d63bcd4ca23218cf5e6